hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -621,8 +625,8 @@ def test_gcsfuse_empty_string_bucket_fails(self):
             j.gcsfuse(self.bucket_name, '')
 
     def test_requester_pays(self):
-        b = self.batch(requester_pays_project='hail-vdc')
-        input = b.read_input('gs://hail-services-requester-pays/hello')
+        b = self.batch(requester_pays_project=PROJECT)
+        input = b.read_input(f'gs://{HAIL_REQUESTER_PAYS_GCS_BUCKET}/hello')
```

Comment:
**Don't do this now**, but the tests will need to have `HAIL_REQUESTER_PAYS_GCS_BUCKET` include the `gs://` scheme. If there's no equivalent of the requester pays in Azure, we'll need to add some sort of decorator to not run this test in Azure.